pkg/agent/endpoints/endpoints.go

```diff
@@ -6,9 +6,11 @@ import (
 	"net"
 	"os"
 
-	sds_v2 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+	discovery_v2 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+	secret_v3 "github.com/envoyproxy/go-control-plane/envoy/service/secret/v3"
 	attestor "github.com/spiffe/spire/pkg/agent/attestor/workload"
-	"github.com/spiffe/spire/pkg/agent/endpoints/sds"
+	"github.com/spiffe/spire/pkg/agent/endpoints/sds/sdsv2"
+	"github.com/spiffe/spire/pkg/agent/endpoints/sds/sdsv3"
```

Comment:
Since these are ultimately different endpoints, it would be nice if we could move them up one level and nuke the common `sds` directory in order to reduce path stutter

Reply:
Sure, happy to do that.